WordPress/Classes/Services/AppUpdate/AppStoreSearchService.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+struct AppStoreLookupResponse: Decodable {
+    let results: [AppStoreInfo]
```

Comment:
(nit) I'd suggest nesting `AppStoreInfo` to avoid extending the global namespace with rarely used types.